src/app/config.cc

```diff
@@ -209,6 +209,19 @@ void Configuration::ParseTunnelsConfig() {
         tunnel.keys = value.get<std::string>(GetAttribute(Key::Keys), "");
         // Parse for CSV destinations + dest:port, then set appropriately
         ParseClientDestination(&tunnel);
+        // Check for conflicting port
+        if (std::find_if(
+                m_TunnelsConfig.begin(),
+                m_TunnelsConfig.end(),
+                [&tunnel](kovri::client::TunnelAttributes const& rhs) {
+                  return tunnel.port == rhs.port;
```

Comment:
Can we please continue to keep naming conventions as specified in ["General Naming Rules"](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules)?